src/chainbin/chain-est-phone-lm.cc

```diff
@@ -32,37 +32,51 @@ int main(int argc, char *argv[]) {
         "Initialize un-smoothed phone language model for 'chain' training\n"
         "Output in FST format (epsilon-free deterministic acceptor)\n"
         "\n"
-        "Usage:  chain-est-phone-lm [options] <phone-seqs-rspecifier> <phone-lm-fst-out>\n"
+        "Usage:  chain-est-phone-lm [options] <phone-seqs-rspecifier-1> [... <phone-seqs-rspecifier-n>] <phone-lm-fst-out>\n"
         "The phone-sequences are used to train a language model.\n"
         "e.g.:\n"
         "gunzip -c input_dir/ali.*.gz | ali-to-phones input_dir/final.mdl ark:- ark:- | \\\n"
         " chain-est-phone-lm --leftmost-context-questions=dir/leftmost_questions.txt ark:- dir/phone_G.fst\n";
 
     bool binary_write = true;
     LanguageModelOptions lm_opts;
+    std::string scales_str;
 
     ParseOptions po(usage);
     po.Register("binary", &binary_write, "Write output in binary mode");
+    po.Register("scales", &scales_str, "comma-separated list of integer valued scale weights used to scale different phone sequences.");
```

Comment:
I think it is a simple change and it is better to keep scales, since in some cases scale as or 100x  and we need to repeat 10 or 100 times, which takes 100x times to process alignments.